New log file opened at Wed Oct 21 15:01:54 CEST 2015 by 7216@3dlab11
Taking next connection to: 54044
New client attempting to connect on port: 54045-54044
  Address = /141.37.31.61:54044
This client on port 54045 has uuid 7d2c862e-6a7a-403d-9f21-56db5d7684ae configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-7d2c862e-6a7a-403d-9f21-56db5d7684ae
Taking next connection to: 54044
New client attempting to connect on port: 54046-54044
  Address = /141.37.31.61:54044
Handshake exception on socket: 54046
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 54044
New client attempting to connect on port: 54047-54044
  Address = /141.37.31.61:54044
This client on port 54047 has uuid 49c6ac08-89f1-4a12-817f-33a436413a42 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-7d2c862e-6a7a-403d-9f21-56db5d7684ae, activator-49c6ac08-89f1-4a12-817f-33a436413a42
Taking next connection to: 54044
New client attempting to connect on port: 54048-54044
  Address = /141.37.31.61:54044
Handshake exception on socket: 54048
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 54044
New client attempting to connect on port: 54049-54044
  Address = /141.37.31.61:54044
This client on port 54049 has uuid 12e21fdc-3dd0-491a-96e1-cd0ee64fecc4 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-49c6ac08-89f1-4a12-817f-33a436413a42, play-fork-run-12e21fdc-3dd0-491a-96e1-cd0ee64fecc4
Taking next connection to: 54044
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54044
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54044
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54044
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 54044
New client attempting to connect on port: 54106-54044
  Address = /141.37.31.61:54044
Handshake exception on socket: 54106
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 54044
New client attempting to connect on port: 54107-54044
  Address = /141.37.31.61:54044
This client on port 54107 has uuid 9611bc88-9154-4bcd-bc41-59602a1f8b7e configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-49c6ac08-89f1-4a12-817f-33a436413a42, play-fork-run-9611bc88-9154-4bcd-bc41-59602a1f8b7e
Taking next connection to: 54044
New client attempting to connect on port: 54131-54044
  Address = /141.37.31.61:54044
Handshake exception on socket: 54131
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 54044
New client attempting to connect on port: 54132-54044
  Address = /141.37.31.61:54044
This client on port 54132 has uuid a327e9dc-963b-4b2a-9d90-a2e399a0da4e configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-49c6ac08-89f1-4a12-817f-33a436413a42, play-fork-run-a327e9dc-963b-4b2a-9d90-a2e399a0da4e
Taking next connection to: 54044
New client attempting to connect on port: 54224-54044
  Address = /141.37.31.61:54044
Handshake exception on socket: 54224
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 54044
New client attempting to connect on port: 54225-54044
  Address = /141.37.31.61:54044
This client on port 54225 has uuid f2bb90db-9e2e-4253-8d66-6460a8f6eae7 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-49c6ac08-89f1-4a12-817f-33a436413a42, play-fork-run-f2bb90db-9e2e-4253-8d66-6460a8f6eae7
Taking next connection to: 54044
